tactics/eqschemes.ml

```diff
@@ -248,17 +248,18 @@ let sym_scheme_kind =
 (**********************************************************************)
 
 let const_of_scheme kind env ind ctx =
-  let sym_scheme, eff = (find_scheme kind ind) in
+  let () = assert (check_scheme kind ind) in
```

Comment:
`lookup_scheme` below will already raise `Not_found` if the scheme is not there, I suggest you actually change the type of `lookup_scheme` to an option type and get rid of the aliases and this check.

Reply:
OK, will do. I have further changes to this API but I'll defer this to another PR for self-containedness though.

Reply:
As you prefer.